components/event-simulator/org.wso2.carbon.event.simulator.ui/src/main/resources/web/eventsimulator/sendConfigValues_ajaxprocessor.jsp

```diff
@@ -23,13 +23,15 @@
 
         JSONObject fileConfigData=new JSONObject(jsonData);
 
-        String fileName=fileConfigData.getString("FileName");
-        String streamID=fileConfigData.getString("streamID");
-        String seperateChar=fileConfigData.getString("seperateChar");
+        String fileName = fileConfigData.getString("FileName");
+        String streamID = fileConfigData.getString("streamID");
+        String seperateChar = fileConfigData.getString("seperateChar");
+        long delayBetweenEventsInMilies = fileConfigData.getLong("delayBetweenEventsInMilies");
 
-        stub.sendConfigDetails(fileName,streamID,seperateChar);
+        stub.sendConfigDetails(fileName, streamID, seperateChar, delayBetweenEventsInMilies);
         msg="Sent";
-    }catch(Exception e){
+    }
+    catch(Exception e){
```

Comment:
Thanks @lasanthafdo, Done in pull request - https://github.com/wso2/carbon-event-processing/pull/160